GOVERNANCE.md

```diff
@@ -8,31 +8,87 @@ This document defines the project governance for Contour.
+* **[contour-operator](https://github.com/projectcontour/contour-operator):** The Contour Operator is a tool for installing and managing Contour.
 
 ## Community Roles
 
+All of the below roles share the responsibility to abide by the Code of Conduct and work together towards making the project better.
+
 * **Users:** Members that engage with the Contour community via any medium (Slack, GitHub, mailing lists, etc.).
 * **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
-* **Maintainers**: The Contour project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
+* **Reviewer:** Review contributions from other members.
+* **Maintainers**: The Contour project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. 
+* **SIG members/owner**: Members of a specific SIG are responsible for the work within that SIG topic, and SIG owners are responsible for driving the efforts within a SIG.
+
+### Reviewer
+Reviewers are able to review code for quality and correctness on some part of a subproject. They are knowledgeable about both the codebase and software engineering principles.
+
+#### Requirements
+
+* Knowledgeable about the codebase
+* Sponsored by a maintainer
+* With no objections from other maintainer
+* New reviewer must be nominated by an existing maintainer or reviewer or self-nominated and must be elected by a supermajority of existing maintainers
+
+#### Responsibilities and privileges
+
+* Code reviewer status may be a precondition to accepting large code contributions
+* Responsible for project quality control via code reviews
+* Focus on code quality and correctness, including testing and factoring
+* May also review for more holistic issues, but not a requirement
+* Expected to be responsive to review requests
+* Assigned PRs to review related to subproject of expertise
+* Assigned test bugs related to subproject of expertise
 
 ### Maintainers
 
 New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
 
+#### Maintainer Expectations
+
+Maintainers have the ability to merge code into the project. Anyone can
```

Comment:
What does "Anyone can become a maintainer" mean specifically? If it's meant to convey that folks from any organization (or no organization at all) are eligible to be maintainers, then I'd say that directly. Otherwise, I'm not sure it really adds anything; the requirements are stated below.